### PR TITLE
Jh links

### PR DESCRIPTION
**Pull request by JEmnetu**
Assigned navbar items to deadlinks so that they would not redirect.

### CODE REVIEW

**Review by kshep425 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -16,45 +16,44 @@
...
+                    <span aria-hidden="true"></span>
+                    <span aria-hidden="true"></span>
+                    <span aria-hidden="true"></span>
+
+                </a>
+            </div>
+            <div id="navbarBasic" class="navbar-menu is active">
+                <div class="navbar-start">
+                    <a href="#" class="navbar-item has-text-grey-darker" id="nutrition_link">Meal Plan</a>
+                    <a href="#" class="navbar-item has-text-grey-dark" id="challenge_link">Challenge a Friend</a>
```
Comment (kshep425) [inline index.html]: Let's just remove links that go nowhere or have them do something.

File: assets/css/style.css [hunk 2]
```diff
@@ -110,12 +119,16 @@ input {
 .logo_column {
     min-width: 50px;
     min-height: 50px;
+}
 
+#new {
```
Comment (kshep425) [inline assets/css/style.css]: What's this for?  Recommend renaming.

File: index.html [hunk 3]
```diff
@@ -16,43 +16,42 @@
...
+                    <span aria-hidden="true"></span>
+                    <span aria-hidden="true"></span>
+                    <span aria-hidden="true"></span>
+
+                </a>
+            </div>
+            <div id="navbarBasic" class="navbar-menu is active">
+                <div class="navbar-start">
+                    <a href="https://nutritionfacts.org" target='_blank' class="navbar-item has-text-grey-darker" id="nutrition_link">Nutrtion Tips & Information</a>
+                    <a href="#" class="navbar-item has-text-grey-dark" id="challenge_link">Challenge a Friend!</a>
```
Comment (kshep425) [inline index.html]: Remove until it does something.

File: index.html (hunk 1 above)
Comment (JEmnetu) [reply index.html]: Corrected links

File: assets/css/style.css (hunk 2 above)
Comment (JEmnetu) [reply assets/css/style.css]: Renamed variable

**Pull request merged**